src/program/lwaftr/run/run.lua

```diff
@@ -124,9 +124,10 @@ end
 -- Requires a V4V6 splitter iff:
 --   Always when running in on-a-stick mode, except if v4_vlan_tag != v6_vlan_tag.
 local function requires_splitter (opts, conf)
-   if not opts["on-a-stick"] then return false end
-   if not conf.vlan_tagging then return true end
-   return conf.v4_vlan_tag == conf.v6_vlan_tag
+   if not opts["on-a-stick"] then return true end
```

Comment:
This inverts the previous behavior, and is not in accordance with the above comment anymore. Who's right, the comment or the code?